### DISCUSSION TITLE
flash build ModBus bridge problem(solved)

### DISCUSSION THREAD

**Discussion by mbu10** [arendst/Tasmota#22391] [General]
good morning 
i have a sdm230 , a fox h1 invertor
i do have a radio link between them using a item, from ebay
But wished to create a WIFI modbus tcp bridge and looked like this can do it the [lilly ](https://github.com/Xinyuan-LilyGO/T-CAN485)
i understand i needed to compile my own this 
i enabled this
#ifndef USE_MODBUS_BRIDGE
#define USE_MODBUS_BRIDGE       // Add support for software Modbus Bridge (+3k code)
#define USE_MODBUS_BRIDGE_TCP   // Add support for software Modbus TCP Bridge (Must also enable USE_MODBUS_BRIDGE)
#endif

  #define SDM230_SPEED         9600              // SDM230-Modbus RS485 serial speed (default: 9600 baud)

and asked to compile in gitpod using  platformio run -e tasmota32
and  installed using the https://tasmota.github.io/install/

when it reboot though in the console just does this 

rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
invalid header: 0x53203131
ets Jul 29 2019 12:21:46
not sure what i have done

thanks martin

**Comment by sfromis** [comment]
Well, what **exactly** did you try to flash? It looks like a wrong file.

**Comment by mbu10** [comment]
I downloaded the .bin file that it's produced for me, and told it to flash that

**Reply by sfromis** [reply]
You did not tell what **exactly** you tried to flash. With ESP32, the file you should use would typically be named something like `tasmota32.factory.bin`

**Reply by mbu10** [reply]
well the tasmo compiler helped me so all good but going to post another problem 
The firsdt compiler did not output a factory file for some reason
